generators/vue/templates/package.json.ejs

```diff
@@ -96,26 +114,17 @@
     "cypress": "<%= nodeDependencies['cypress'] %>",
 <%_ } _%>
     "rimraf": "<%= nodeDependencies['rimraf'] %>",
+    "rollup-plugin-copy": null,
```

Comment:
it's mandatory?

Reply:
I’ve added a transform that replaces null values with the value from nodeDependencies.
Maybe this is too confusing?

Reply:
ok a little, but ok